src/app/Traits/DbUniqueIdentifier.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace NetworkRailBusinessSystems\UserLogin\Traits;
+
+trait DbUniqueIdentifier
+{
+    /**
+     * This trait method retrieve the unique identifier (such as a GUID) for a user from the local database
+     */
+    public static function uniqueIdentifier(string $username): ?string
+    {
+        // Model for the local database
```

Comment:
These comments can already be inferred from the code itself, they are not needed.

Reply:
removed all from both traits